androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java

```diff
@@ -543,20 +581,20 @@ private void setFields (String connectionString, String username, String passwor
      */
     private void checkStatusMessage(){
 
-        switch (trusted){
-            case (-1):
+        switch (servertype.getStatus()){
```

Comment:
An enum can be used directly in a ```switch``` statement. You don't need to get its internal value, just write

```switch (servertype)```

and then the cases as

```case HTTP:```,  etc...